site/content/docs/5.0/utilities/opacity.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: docs
+title: Opacity
+description: Use opacity utilities to quickly add transparency to the background of an element.
+group: utilities
+toc: true
+---
+
+## Opacity
+
+Opacity classes are available, though they are not responsive.
```

Comment:
Opacity responsive. What does that mean 🤔  ?

Reply:
means it has no opacity break point, opacity-sm-10, opacity-md-20, etc